data/fwupd.conf

```diff
@@ -0,0 +1,5 @@
+# use `man 5 fwupd.conf` for documentation
+[fwupd]
+DisabledPlugins=test;test_ble
+OnlyTrusted=true
+AllowEmulation=false
```

Comment:
So i'm worried that by us not putting default values into this file or into the man page it's going to be impossible for anyone to know what the default behavior actually is when nothing is there.  Sure they can look at the code a source of truth, but I'm really suspecting we're going to want to have the default values stored SOMEWHERE that ends up in the code and in the documentation.  It could be a macro with a fu-engine-conf-defaults.h or something that is processed.

Reply:
Yup, I think that's true, but not for this PR I think.

Reply:
How about this: https://github.com/fwupd/fwupd/pull/5730